README.md

```diff
@@ -293,6 +294,7 @@ defaults):
 | `spel_epel7release`     | URL to the release RPM for the [EPEL 7][10] repo                  |
 | `spel_epelrepo`         | Name of the epel repo (if different than "epel")                  |
 | `spel_extrarpms`        | Comma-separated list of extra package/@group names to pass to yum |
+| `spel_fstype`           | Filesystem-type to use for root filesystems. Default is `ext4`. Other supported options are `ext3` and `xfs` |
```

Comment:
It would be nice to maintain column consistency in the table. Either align all the columns, or disalign all of them...

Reply:
Yeah. I _wanted_ to maintain column-width consistency, but GitHub-flavored markdown doesn't seem to provide a method for doing so?

Reply:
Alright. Pushed a change: columns are aligned ...even if the editor window needs to be 220 columns wide to prevent linewrapping in editor window. :stuck_out_tongue: